docs/components/bottom_sheets.md

```diff
@@ -0,0 +1,286 @@
+                      return Wrap(
+                        children: [
+                          ListTile(
+                            title: Text(
+                              'Header',
+                              style: theme.textTheme.subtitle1
+                                  .copyWith(color: theme.colorScheme.onPrimary),
+                            ),
+                            tileColor: theme.colorScheme.primary,
+                          ),
+                          ListTile(
+                            title: Text('Title 1'),
+                          ),
+                          ListTile(
+                            title: Text('Title 2'),
+                          ),
+                          ListTile(
+                            title: Text('Title 3'),
+                          ),
+                          ListTile(
+                            title: Text('Title 4'),
+                          ),
+                          ListTile(
+                            title: Text('Title 5'),
+                          ),
+                        ],
+                      );
+                    },
+                  );
+                },
+              ),
+            );
+          },
+        ),
+      ),
+    );
+  }
+}
+```
+
+The persistent bottom sheet can be used for a standard bottom sheet.
+Use a [DraggableScrollableSheet](https://api.flutter.dev/flutter/widgets/DraggableScrollableSheet-class.html) for more custom dragging and snap points.
+
+### Anatomy and key properties
+
+The following shows the anatomy of a standard bottom sheet:
+
+![Standard bottom sheet anatomy](assets/bottom_sheets/standard-anatomy.png)
+
+1. Sheet
+1. Contents
+
+#### Sheets properties
+
+&nbsp;         | Property
+-------------- | ------------------------
+| **Color** | `backgroundColor` |
+| **Elevation** | `elevation` |
+| **Shape** | `shape` |
+
+#### Contents properties
+
+There are no specific properties for content because the content can be any composition of widgets.
+
+## Modal bottom sheet
+
+Modal bottom sheets present a set of choices while blocking interaction with the rest of the screen. They are an alternative to inline menus and simple dialogs on mobile, providing additional room for content, iconography, and actions.
+
+Modal bottom sheets are used in mobile apps only.
+
+### Modal bottom sheet example
+
+`BottomSheet`
+- [Class definition](https://api.flutter.dev/flutter/material/showModalBottomSheet.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/bottom_sheet.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/bottom_sheets/modal&gh_ref=develop)
+
+The following is an example modal bottom sheet:
+![A modal bottom sheet with share, link, edit, and delete list items](assets/bottom_sheets/SheetsBottom_modal_screenshot.jpg)
```

Comment:
Suggested alt-text:

Bottom sheet with share, link, edit, and delete items partially covering screen image with scrim of dogs drinking water.